components/controls/buttons.css

```diff
@@ -52,6 +52,15 @@
   box-shadow: 0 0 0 3px rgba(35, 106, 151, 0.25);
 }
 
+.button-primary {
+  background: var(--darkBlue);
+  color: #fff;
+}
+
+.button-primary:hover {
```

Comment:
Should we also add a prop in the button component to toggle this ? So we don't have to know classes and mess with them too much when using a button ?

Reply:
I was thinking about that, yeah. Same for the `button-red` class, which is used all over the place. But I wasn't sure how to implement it, and this being kinda urgent, I went for the quick win 😄 .

Reply:
Yeah fine with the quick win. Anyway I think the goal at some point will be to rewrite this whole buttons module to use emotion.